spec/mail_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'ostruct'
+
+unless defined?(Devise::Mailer)
+  module Devise
+    class Mailer
+      def self.test_message(_)
+        OpenStruct.new(deliver_later: :ok)
+      end
+    end
+  end
+end
```

Comment:
I take back this comment. Please delete this file entirely. This is supposed to be stubbing a method, but as soon as this is loaded, the stub is active for every test. Instead, you should stub this in a `before` block in the spec file:
```ruby
before do
  allow(Devise::Mailer).
    to receive(:test_message).
    and_return(OpenStruct.new(deliver_later: :ok))
end
```